### PR TITLE
Add management support.

### PR DESCRIPTION
**Pull request by jmjoy**
1. Add report instance properties support.
2. Add keep alive support.
3. Replace function params `ToString` to `Into<String>` for better semantics.
4. Upgrade min rust version requirement.

### CODE REVIEW

**Review by wu-sheng — DISMISSED** [review]

**Comment by wu-sheng** [comment]
Do the properties of instance support customization? Such as using a JSON to inject some flags to instance properties?

File: src/management/instance.rs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+    pub fn update(&mut self, key: &str, value: impl Into<String>) {
+        if let Some(values) = self.inner.get_mut(key) {
+            *values = vec![value.into()];
+        }
+    }
+
+    pub fn remove(&mut self, key: &str) {
+        self.inner.remove(key);
+    }
```
Comment (jmjoy) [inline src/management/instance.rs:92]: @wu-sheng These methods is designed for user to custom the properties.

Comment (wu-sheng) [reply src/management/instance.rs:92]: Then Let's document it on the readme.

Comment (wu-sheng) [reply src/management/instance.rs:92]: Including how to customize and what are default tag keys and values.

**Review by wu-sheng — APPROVED** [review]

File: src/management/instance.rs (hunk 1 above)
Comment (jmjoy) [reply src/management/instance.rs:92]: Done.

**Pull request merged**